mapclientplugins/datamapstep/model/mapper.py

```diff
@@ -66,9 +67,9 @@ def update_data_coordinates_field(self, field_name):
         with ChangeManager(self._field_module):
             data_points = self._field_module.findNodesetByFieldDomainType(Field.DOMAIN_TYPE_DATAPOINTS)
 
-            self._active_data_point_group_field = self._field_module.createFieldNodeGroup(data_points)
+            field_group = self._field_module.createFieldGroup()
             tmp_true = self._field_module.createFieldConstant([1])
-            active_datapoints_group = self._active_data_point_group_field.getNodesetGroup()
+            active_datapoints_group = field_group.createNodesetGroup(data_points)
```

Comment:
@rchristie, could you please have a quick look at this one when you get a chance. Is this the correct replacement if we were previously calling `createFieldNodeGroup` directly on the `Fieldmodule`?

Reply:
Yes. The MeshGroup/NodesetGroups must be explicitly created within the group. A group can have all sorts of things in it and we don't automatically populate it with subobject groups.
In the above code the FieldNodeGroup was made as a standalone field without being managed by a field group, which is no longer supported (or even exists); with the subobject-specific group field, you were guaranteed to be able to 'get' a NodesetGroup from it, but not from a generic Group with the new way.